src/mb_caa_dimensions/style.tsx

```diff
@@ -0,0 +1,8 @@
+import USERSCRIPT_NAME from 'consts:userscript-name';
+import css from './style.scss';
+
+export function setupStyle(): void {
+    document.head.append(<style id={'ROpdebee_' + USERSCRIPT_NAME}>
+        {css}
+    </style>);
+}
```

Comment:
These style setup functions will probably appear in all userscripts sooner or later and a similar function also exists for the update notifications. On the one hand this cries for a shared function in `@lib`, on the other hand they are fairly trivial.

Reply:
Yeah, I guess a function in `@lib` would make sense, but let's wait for #458 again so we can use that `@lib` function there too.